homeassistant/components/dynalite/bridge.py

```diff
@@ -61,6 +70,26 @@ def update_device(self, device: Optional[DynaliteBaseDevice] = None) -> None:
         else:
             async_dispatcher_send(self.hass, self.update_signal(device))
 
+    @callback
+    def handle_notification(self, notification: DynaliteNotification) -> None:
+        """Handle a notification from the platform and issue events."""
+        if notification.notification == NOTIFICATION_PACKET:
+            self.hass.bus.fire(
+                f"dynalite_{self.host}_packet",
```

Comment:
It's probably better to put the host in the event data and have a static event type. The trigger is not templatable in an automation so the user can't use a template to filter for host in the trigger.

Reply:
thanks. i assumed that there is some standard but couldn't find it. That's why i waited with the documentation to see where we land. will fix quickly

Reply:
fixed